pumpkin/src/command/commands/cmd_pumpkin.rs

```diff
@@ -9,7 +9,7 @@ use crate::{
     server::CURRENT_MC_VERSION,
 };
 
-const NAMES: [&str; 1] = ["pumpkin"];
+const NAMES: [&str; 2] = ["pumpkin", "version"];
```

Comment:
I think /pumpkin is enough

Reply:
/version is more intuitive and generic, its implemented like this in bukkit/spigot/paper. Do i still remove it ?

Reply:
Oh, was did not know that Bukkit uses that